src/pages/home/sidebar/OptionRow.js

```diff
@@ -129,12 +129,9 @@ const OptionRow = ({
                                         avatarImageURLs={option.icons}
                                         optionIsFocused={optionIsFocused}
                                         size={mode === 'compact' ? 'small' : 'default'}
-                                        styles={hovered && !optionIsFocused && {
-                                            secondAvatar: {
-                                                backgroundColor: themeColors.sidebarHover,
-                                                borderColor: themeColors.sidebarHover,
-                                            },
-                                        }}
+                                        secondAvatarStyle={hovered && !optionIsFocused
```

Comment:
Sorry, I'm a little unsure still about whether it's right to pass this style as a prop instead of defining it inside `MultipleAvatars`.

Can we pass a `hovered` prop (default `false`) to `MultipleAvatars` and change here

https://github.com/Expensify/Expensify.cash/blob/36ecd1080e972880d4d753a076e3a35058ee5091/src/components/MultipleAvatars.js#L49-L53

so that 

```js
    const secondAvatarStyles = [
        size === 'small' ? styles.secondAvatarSmall : styles.secondAvatar,
        optionIsFocused ? styles.focusedAvatar : styles.avatar,
        hovered && !optionIsFocused ? styles.secondAvatarHovered : {},
    ];
```

Reply:
I passed the style prop as MultipleAvatar is being used on multiple location where background color could be different. `secondAvatar` is just used to show a border around the second avatar. so we may need to have adjust the style of border.

Reply:
Check it here https://github.com/Expensify/Expensify.cash/pull/2104/commits/8c6b211c4c579bd3119b1fda2019cf957fad24d5

Reply:
@marcaaron any thoughts on this?

Reply:
Is it not possible to just use a transparent background and border color?

Reply:
Seems we are doing a lot to match the background of one component to the parent.

Reply:
Transparent background and border color do not work. The white border around the second avatar is not visible. So there are hovered and focused states for the OptionRow. To match all these, I think the only possible way, AFAIK, is to change the styling which is absolutely fine. Allowing parents to pass the style is fine is easiest.

Reply:
@marcaaron. Do you still think, we can improve something?. If this looks justifying, I think we are good to merge this so that two linked issues will be resolved as well. Otherwise, let me know, I will prepare a separate PR for other issues.

Reply:
>Allowing parents to pass the style is the easiest.

Hmm sorry, I still think this is a strange implementation. The problem here is that it's not obvious how to use this component prop in other places. e.g. if the behavior we want is to provide an optional hover style then we should maybe do this:

- `isHovered` prop
- `secondAvatarHoverStyle`

Then control whether to use the hover style in `MultipleAvatars`. I think this will be more instructive when others try to re-use this component.

Reply:
I see. Let me have a look at this.

Reply:
Now, I am uncertain how to match the styling of the border on the Search Page. OptionRow is used on Search Page and LHN and both have different backgrounds. Thus the Styling for MultipleAvatars will change. I think creating a set of themes would be fine. we can pass a prop `theme` which will take `LHN` or `default` etc values. and then we can adjust the style based on theme value.

Reply:
What about passing a `backgroundColor` + `isHovered` prop?

Reply:
A Few details:
we have to manage the Border around the secondAvatar. We want to match the background color. There are three states for OptionRow. 
a) default.
b) Hovered
c) Focused.

Now option rows theme is different for LHN and Search Page. it means a different set of colors for the above-mentioned states.

MultipleAvatar is also used on Header where we are only concerned with the `default` state but the color is different from `OptionRow`.

As per the above comment, We would be needing.
```
                                      <MultipleAvatars
                                        avatarImageURLs={option.icons}
                                        optionIsFocused={optionIsFocused}
                                        isHovered={}
                                        secondAvatarHoveredStyle={}
                                        secondAvatarFocusedStyle={}
                                    />
          ```